dfm_backend/app.py

```diff
@@ -0,0 +1,322 @@
+from datetime import UTC, datetime, timedelta
+from flask import Flask, request, jsonify, send_from_directory
+from flask_cors import CORS
+from flask_limiter import Limiter
+from flask_limiter.util import get_remote_address
+from flask_mail import Mail, Message
+import json
+from os import getenv
+from pathlib import Path
+from typing import Any
+import uuid
+
+def create_app(config: dict[str, Any] | str | Path | None = None) -> Flask:
+    if config is None:
+        config = getenv('APP_CONFIG', Path(__file__).resolve().parent / 'app.conf.py')
+    if isinstance(config, (str, Path)):
+        config_path = Path(config).absolute()
+        if not Path(config_path).exists():
+            raise FileNotFoundError(f"Config file {config_path} does not exist.")
+        config = {}
+        exec(open(config_path).read(), {}, config)
+        local_config_path = config_path.with_suffix('.local.py')
+        if Path(local_config_path).exists():
+            exec(open(local_config_path).read(), {}, config)
+    app = Flask(__name__, static_folder=config['STATIC_FOLDER'], static_url_path='')
+    app.config.update(config)
+    limiter = Limiter(app=app, key_func=get_remote_address)
+    mail = Mail(app)
+    CORS(app)
+
+    TOKENS = {}
+
+    def load_news():
+        if not Path(app.config['NEWS_FILE']).exists():
+            return []
+        with open(app.config['NEWS_FILE']) as f:
+            return [json.loads(line) for line in f]
+
+    def save_news(item):
+        with open(app.config['NEWS_FILE'], 'a') as f:
+            f.write(json.dumps(item) + '\n')
+            
+    @app.route('/api/news', methods=['GET'])
+    def get_news():
+        return jsonify(load_news())
+
+    @app.route('/api/news', methods=['POST'])
+    def post_news():
+        auth = request.headers.get('Authorization', '')
+        token = auth.replace('Bearer ', '')
+        token_expiry = TOKENS.get(token, None)
```

Comment:
Nit: None is default.

```suggestion
        token_expiry = TOKENS.get(token)
```